2019-07-30 21:48:23,989 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:75] - DEBUG: debug
2019-07-30 21:48:23,989 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:76] - INFO: info
2019-07-30 21:48:23,989 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:77] - WARNING: 警告
2019-07-30 21:48:23,989 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:78] - ERROR: 报错
2019-07-30 21:48:23,989 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:79] - CRITICAL: 严重

------------Following Log was Generated By a New Logger------------

2019-07-30 21:48:29,656 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:75] - DEBUG: debug
2019-07-30 21:48:29,666 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:76] - INFO: info
2019-07-30 21:48:29,666 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:77] - WARNING: 警告
2019-07-30 21:48:29,666 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:78] - ERROR: 报错
2019-07-30 21:48:29,666 - all.log - D:/Win/Doc/#Project/RFML/python/my_py_tools/my_logger.py[line:79] - CRITICAL: 严重
